clients/vscode-hlasmplugin/src/test/syntaxes/endevor/mix.listing.snap

```diff
@@ -641,19 +641,18 @@
 #^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
 #                                                        ^^^ source.hlasmListing punctuation.definition.tag.hlasm
 #                                                           ^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing
-#                                                                                    ^^^^^^^^^^^^^^^^ source.hlasmListing comment.hlasm
-#                                                                                                    ^^^^^^^^^^^^ source.hlasmListing
+#                                                                                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.hlasmListing comment.hlasm
```

Comment:
Strictly speaking, the comment starts right after the space after the `OP2` operannd.